tests/src/test/scala/system/basic/WskSequenceTests.scala

```diff
@@ -216,7 +216,7 @@ class WskSequenceTests
                     result.fields.get("error") shouldBe Some(JsString(sequenceIsTooLong))
                     // check that inner sequence had only (limit - 1) activations
                     val innerSeq = activation.logs.get(1) // the id of the inner sequence activation
-                    val getInnerSeq = wsk.activation.get(innerSeq)
+                    val getInnerSeq = wsk.activation.get(Some(innerSeq))
```

Comment:
hmm seems wrapping all ids with Some is too common - should there be an overloaded `get` instead of changing all the test cases?